docs/docs/adrs/adr-007-pause-unbonding-on-eq-prop.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 2
+title: ADR Template
+---
+# ADR 007: Pause validator unbonding during equivocation proposal
+
+## Changelog
+* 2023-05-11: Initial Draft
+
+## Status
+
+Proposed
+
+## Context
+
+(Copied from #747)
+
+Currently, if an equivocation slashing proposal is created after more than one
+week has passed since the equivocation, it is possible that the validator in
+question could unbond and get away without being slashed, since the unbonding
+period is 3 weeks, and the voting period is 3 weeks. For this reason, it might
+be good to pause unbondings for validators named in an equivocation slashing
+proposal until the proposal's voting period is over.
+
+## Decision
+
+### How
+
+Pausing the unbonding period is already possible thanks to the changes in the
+staking module of the cosmos-sdk:
+- `stakingKeeper.PutUnbondingOnHold` pauses an unbonding period
+- `stakingKeeper.UnbondingCanComplete` unpauses an unbonding period
+
+Those methods use a counter under the hood, which means we can use them without 
+conflicts with the *Completion of Unbonding Operations* system. Giving an
+unbonding period (already paused by definition because of the *Completion of
+Unbonding Operations* system), an additional pause will just increase the
+counter, so when this unbonding period has reached its maturity on all provider
+and consumer chain, it will remain paused.
+
+### When pause
+
+The unbonding period (if there's any) should paused once an equivocation
+proposal is activated (in other words, when it enters the voting period). For
+that, the `gov` module hooks `AfterProposalDeposit` can be used. 
+
+If the proposal is an equivocation proposal in voting period for the first time
+(this is important because a proposal can receive additional deposit, even
+after its activation), then for each equivocation of the proposal, the related
+unbonding operations of the related validator must be paused.
+
+### When unpause
+
+Similarly, a `gov` module hook can be used, here it's
+`AfterProposalVotingPeriodEnded`.
+
+If the proposal is an equivocation proposal, then for each equivocation,
+unpause all unbonding operations of the related validator.
+
+## Consequences
+
+### Positive
+
+- Validators threatened by an equivocation proposal cannot play with the time to
```

Comment:
The meaning it's clear but I would rephrase slightly, `cannot play with the time to..` in particular should be rewritten in my opinion

Reply:
rephrased thx!